stock_inventory_discrepancy/models/stock_inventory_line.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class StockInventoryLine(models.Model):
+    _inherit = 'stock.inventory.line'
+
+    @api.one
+    def _compute_discrepancy(self):
+        self. discrepancy_qty = self.product_qty - self.theoretical_qty
+        if self.theoretical_qty:
+            self.discrepancy_percent = 100 * (abs(
+                self.product_qty - self.theoretical_qty)) \
+                / self.theoretical_qty
```

Comment:
Still there's a problem of what you do when the theoretical quantity is 0, and you report a different quantity. In that case the percent of discrepancy would be infinite, but you are now assuming 0. Perhaps in that case we should put 999%??